### DISCUSSION TITLE
What is the proper support channel for the paid version ?

### DISCUSSION THREAD

**Discussion by pixeline** [livewire/flux#297] [Feature Requests] [answered]
I have an issue setting up the paid version of fluxui. I see that the project's repository only allows bug features and feature requests. So I've sent a message to Caleb on Xitter, as well as an email.  Could you clarify what is the support channel paid users are supposed to use ? 
Thanks!

**Comment by joshhanley** [comment, marked as answer]
@pixeline if it's a technical problem, raise an bug issue on this repo, if it's an account/ billing issue then send an email to support@fluxui.dev

**Reply by pixeline** [reply]
OK, will do!
